sdk/appconfiguration/app-configuration/test/policies/throttlingRetryPolicyTests.spec.ts

```diff
@@ -12,7 +12,7 @@ import {
   RestError
```

Comment:
GitHub isn't showing a file rename here. Just curious why this isn't under `internal` or `public`.

Reply:
Thanks. Moved it to internal